utils/config-schema.json

```diff
@@ -145,7 +145,7 @@
         "MakerRegistry",
         "DexRegistry"
       ],
-      "additionalProperties": false
+      "additionalProperties": true
```

Comment:
That's because of ArgentWalletDetector currently present in the config files on AWS. Alternatives that I see are:
- to explicitly add ArgentWalletDetector to the schema without adding the corresponding contracts (inconsistent)
- to explicitly add ArgentWalletDetector to the schema and add the corresponding contract (scope creep)
- to revert PR #156 and remove ArgentWalletDetector from the config files